source/LoadPanel.cpp

```diff
@@ -416,6 +416,10 @@ void LoadPanel::UpdateLists()
 	for(const string &path : fileList)
 	{
 		string fileName = Files::Name(path);
+		// Skip the Steam Auto Cloud file, which is created by Steam and not a real save file.
```

Comment:
```suggestion
		// Skip the Steam Auto Cloud file, which is created by Steam but not a real save file.
```

I think this may be a bit more apt?